src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/EmptyCatchBlock/case2/context/Test.java

```diff
@@ -0,0 +1,45 @@
+package TreeWalker.blocks.EmptyCatchBlock;
+
+import java.io.IOException;
+
+public class Test {
+    private void foo() {
+        try {
+            throw new RuntimeException();
+            // violation context at next line
+        } catch (Exception expected) {
+        }
+    }
+
+    private void foo1() {
+        try {
+            throw new RuntimeException();
+            // violation patchedline at next line
+        } catch (Exception e) { }
+
+    }
+
+    private void foo2() {
+        try {
+            throw new IOException();
+            // violation patchedline at next line
```

Comment:
violation from `context` is expected, not from `patchedline`

Reply:
done